modules/simple-payments/simple-payments.php

```diff
@@ -241,7 +239,7 @@ function output_shortcode( $data ) {
 			esc_attr( "${css_prefix}-description" ),
 			$data['description'],
 			esc_attr( "${css_prefix}-price" ),
-			esc_html( $data['price'] ),
+			$data['price'], // Escaped by format_price
```

Comment:
Should we move `esc_html` here just so that we do it as late as possible before output, to be clear that we're really doing it?

Reply:
That's an important question. I followed some existing patterns that produce appropriately escaped html for this bit. It results in something like `<abbr title="United States Dollars">$</abbr> 42.99`. If we escape here, that doesn't turn out well 🙂 

I believe that's a change from the existing implementation and I'm not sure whether it's desired. If we just want to produce `$ 42.99`, we can continue to escape here and everything's simpler.

@thomasguillot Thoughts?

Reply:
Oh right, that makes sense then. I didn't test run this PR yet, just followed my gut feeling of _better escape late_.

I'll leave it to @thomasguillot if `$` is enough or if it needs `<abbr>` :-)

Reply:
>Also in that case it's a misuse of the `abbr` tag because `<abbr title="United States Dollars">` should be around `USD` and not `$`. Anyway... like I said, it's not needed 😊

Valid point! :thumbsup:

> the sign for SGD should be S$, just like C$ for CAD, etc...

Gotcha, we can fix this.

Reply:
Singapore dollar $ follow-ups here: https://github.com/Automattic/wp-calypso/pull/28380